TCMSApp/src/client/app/layout/shell.html

```diff
@@ -1,7 +1,7 @@
-<div ng-controller="ShellController as vm">
-    <ht-top-nav navline="vm.navline"></ht-top-nav>
+<div ng-controller="ShellController as vm" in-init="showShell">
```

Comment:
can we create `activate` function in the controller with this init (and something else)